.gitignore

```diff
@@ -1,13 +1,13 @@
-/target
-/local
+/**/target
```

Comment:
@ge0ffrey That was my modification. I fixed that by adding .gitignore file to root directories of  mortgages and optacloud examples.